service/eventer/github/github.go

```diff
@@ -56,6 +52,7 @@ type Eventer struct {
 
 	// Internals.
 	bootOnce sync.Once
+	etagMap  map[string]string
```

Comment:
The etag map is now shared for all API actions. This prevents the rate limit to be killed after a couple of minutes.